### DISCUSSION TITLE
Welcome to pollitup Discussions!

### DISCUSSION THREAD

**Discussion by agamjotsingh18** [agamjotsingh18/pollitup#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by agamjotsingh18** [comment]
Tweet about your first contribution experience by tagging me(@_agamjotsingh). Would love to retweet.

**Comment by ronakjain2012** [comment]
Hi, I have checked the MVP and seems a good idea

Correct me if wrong 
This project is for location based Polls where we could check the nearby polls and vote and that data could help the local community to make decisions.
